cv_next/app/feed/components/CVItemRSC.tsx

```diff
@@ -1,44 +1,53 @@
-import Image from "next/image"
+import Image, { StaticImageData } from "next/image"
 import Link from "next/link"
 import generateImageUrl from "@/helpers/imageURLHelper"
 import Categories from "@/types/models/categories"
-import { ClientCvModel } from "@/types/models/cv"
+import { useState } from "react"
```

Comment:
Note that this is supposed to be a static, server side component. Using hooks is not allowed in server components (and I'm pretty sure you'd get at least a warning about it).
Render a placeholder image here (that  "blank" img), and move the actual stateful render down to the CvItem.tsx file

Reply:
Specifically, this file is updated in a different branch that updates the preview source to be supabase so this is outdated

Reply:
BTW - outdated files that are updated in different branches - we'll merge this (big) branch and the other branches would resolve conflicts for themselves (and not, for example, merging a side-branch to this big branch).

Reply:
So in this specific case wouldn't it be better to add the changes from there if they're ready?

Reply:
Nope. This'll only add more complications to this already enormous branch. Merge this branch and only then open PR to the other branch. There's absolutely nothing wrong with resolving conflicts.

Reply:
Ok
Updated the change requested at the start of the thread